shared/js/tab.js

```diff
@@ -122,14 +129,4 @@ class Tab {
     }
 }
 
-chrome.webRequest.onBeforeRedirect.addListener((req) => {
-    // count redirects
-    let tab = tabManager.get({'tabId': req.tabId})
-    if (!tab) return
-
-    if (tab.httpsRedirects[req.requestId]) {
-        tab.httpsRedirects[req.requestId] += 1
-    } else {
-        tab.httpsRedirects[req.requestId] = 1
-    }
-}, {urls: ["*://*/*"]})
```

Comment:
Now, I know I said I was not going to move the `chrome` event listeners, but I was having trouble with this for some reason. `tabManager.get()` was erroring out as "not a function". It got fixed when I moved this chunk to `tabManager.js`.

Possibly something to do with the circular dependency between the two? I'm not sure.